views/js/qtiCreator/widgets/interactions/extendedTextInteraction/states/Question.js

```diff
@@ -139,6 +141,20 @@ define([
             interaction.attr('patternMask', attrValue);
         };
 
+        callbacks.expectedLength = function(interaction, attrValue){
+            var newValue = parseInt(attrValue,10);
+            if(! isNaN(newValue)){
+                interaction.attr('expectedLength', attrValue);
+            }
+        };
```

Comment:
As we don't really have a good way to do it, Can I suggest to discuss that point later an consider this as ok for this merge ?